scripts/dev.js

```diff
@@ -184,13 +185,30 @@ class ActionServer extends BaseScript {
         resources.dummyProc = execa('node')
       }
       this.emit('progress', 'press CTRL+C to terminate dev environment')
+
+      if (hasBackend) {
+        // fetch action logs
+        const AppScripts = require('../index.js')
+        const scripts = AppScripts({ listeners: {} })
+        this.getLogs(scripts.logs, this.getLogs)
+      }
     } catch (e) {
       aioLogger.error('Unexpected error. Cleaning up.')
       cleanup(e, resources)
     }
     return frontEndUrl
   }
 
+  async getLogs (logScript, cb) {
+    try {
+      await logScript([], {})
+    } catch (e) {
+      aioLogger.error('Error while fetching action logs ' + e)
+    } finally {
+      setTimeout(function () { cb(logScript, cb) }, fetchLogInterval)
```

Comment:
what if there were 5 invocations in between two fetch log interval?

Reply:
I have made a few changes to log function. Now we fetch default 30 activations but keep track of last activation which was logged. This way we should not miss any activations, plus we don't end up repeat logging them.